### PR TITLE
NMS-14436 Enlinkd Clean Adapter Classes

### PR DESCRIPTION
**Pull request by rssntn67**
* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-14436

These changes are anly syntctic changes.
They have been done for next steps in which the code will be improved.

### CODE REVIEW

**Comment by rssntn67** [comment]
Please take a look at the changes. 
In particular you should check synchronization statement I changed.

File: features/enlinkd/adapters/common/src/main/java/org/opennms/netmgt/enlinkd/common/TopologyUpdater.java [hunk 1]
```diff
@@ -75,10 +75,11 @@ public static OnmsTopologyVertex create(NodeTopologyEntity node, IpInterfaceTopo
...
     private static final Logger LOG = LoggerFactory.getLogger(TopologyUpdater.class);
 
-    private OnmsTopologyDao m_topologyDao;
-    private NodeTopologyService m_nodeTopologyService;
-    private TopologyService m_topologyService;
+    private final OnmsTopologyDao m_topologyDao;
+    private final NodeTopologyService m_nodeTopologyService;
+    private final TopologyService m_topologyService;
 
+    private final Object m_lock= new Object();
```
Comment (christianpape) [inline features/enlinkd/adapters/common/src/main/java/org/opennms/netmgt/enlinkd/common/TopologyUpdater.java:82]:
```suggestion
    private final Object m_lock = new Object();
```

File: features/enlinkd/daemon/src/main/java/org/opennms/netmgt/enlinkd/EnhancedLinkd.java [hunk 2]
```diff
@@ -42,8 +42,7 @@
 import org.opennms.netmgt.config.datacollection.SnmpCollection;
 import org.opennms.netmgt.daemon.AbstractServiceDaemon;
 import org.opennms.netmgt.enlinkd.api.ReloadableTopologyDaemon;
-import org.opennms.netmgt.enlinkd.common.NodeCollector;
-import org.opennms.netmgt.enlinkd.common.TopologyUpdater;
+import org.opennms.netmgt.enlinkd.common.*;
```
Comment (christianpape) [inline features/enlinkd/daemon/src/main/java/org/opennms/netmgt/enlinkd/EnhancedLinkd.java:45]: Please use single import statements.

File: features/enlinkd/service/impl/src/main/java/org/opennms/netmgt/enlinkd/service/impl/BridgeTopologyServiceImpl.java [hunk 3]
```diff
@@ -73,16 +73,17 @@ public class BridgeTopologyServiceImpl extends TopologyServiceImpl implements Br
...
+    private final Object lock = new Object();
     private BridgeElementDao m_bridgeElementDao;
     private BridgeBridgeLinkDao m_bridgeBridgeLinkDao;
     private BridgeMacLinkDao m_bridgeMacLinkDao;
     private BridgeStpLinkDao m_bridgeStpLinkDao;
     private IpNetToMediaDao m_ipNetToMediaDao;
 
-    volatile Map<Integer, Set<BridgeForwardingTableEntry>> m_nodetoBroadcastDomainMap= new HashMap<>();
-    volatile Set<BroadcastDomain> m_domains;
+    private final Map<Integer, Set<BridgeForwardingTableEntry>> m_nodetoBroadcastDomainMap= new HashMap<>();
```
Comment (christianpape) [inline features/enlinkd/service/impl/src/main/java/org/opennms/netmgt/enlinkd/service/impl/BridgeTopologyServiceImpl.java:84]:
```suggestion
    private final Map<Integer, Set<BridgeForwardingTableEntry>> m_nodetoBroadcastDomainMap = new HashMap<>();
```

**Review by christianpape — CHANGES_REQUESTED** [review]
Only very small fixes needed.

**Review by christianpape — APPROVED** [review]

**Pull request merged**